docs/en/verifying-signatures.wml

```diff
@@ -9,199 +9,380 @@
     <hr>
-    <p>Each file on <a href="<page download/download>">our download
+    <p>
+    Each file on <a href="<page download/download>">our download
     page</a> is accompanied by a file with the same name as the
     package and the extension ".asc". These .asc files are GPG
     signatures. They allow you to verify the file you've downloaded
     is exactly the one that we intended you to get. For example,
     torbrowser-install-<version-torbrowserbundle>_en-US.exe is accompanied by
     torbrowser-install-<version-torbrowserbundle>_en-US.exe.asc. For a list
-    of which developer signs which package, see our <a href="<page docs/signing-keys>">signing keys</a> page.</p>
-
-    <p>We now show how you can verify the downloaded file's digital signature on
-    different operating systems. Please notice that a signature is dated the moment
-    the package has been signed. Therefore every time a new file is uploaded a new
-    signature is generated with a different date. As long as you have verified the
-    signature you should not worry that the reported date may vary.
+    of which developer signs which package, see our
+    <a href="<page docs/signing-keys>">signing keys</a> page.
     </p>
-
-    <h3>Windows</h3>
-    <hr>
-    <p>First of all you need to have GnuPG installed before you can verify signatures.
-    Download it from <a href="https://gpg4win.org/download.html">https://gpg4win.org/download.html</a>.</p>
-    <p>Once it's installed, use GnuPG to import the key that signed your
+  </label>
+  <article class="ac-box">
+
+   <p><span class="hint">Choose your operating system:</span>
+   <label for="ac-2-1"><a name="KeyWindows">&#9658; Windows</a></label>
+   <label for="ac-2-2"><a name="KeyMacOS">&#9658; Mac OS X</a></label>
+   <label for="ac-2-3"><a name="KeyLinux">&#9658; Linux</a></label>
+   </p>
+<!-- Windows -->
+  <div>
+   <input id="ac-2-1" name="accordion-2" type="radio" checked />
+   <article class="ac-os">
+    <h3>Import OpenPGP key on Windows</h3>
+    <p>
+    First of all you need to have GnuPG installed before you can verify
+    signatures.
+    <br/>
+    Download it from
+    <a href="https://gpg4win.org/download.html">https://gpg4win.org/download.html</a>.
+    </p>
+    <p>
+    Once it's installed, use GnuPG to import the key that signed your
     package. In order to verify the signature you will need to type a few commands
     in windows command-line, <i>cmd.exe</i>.
-    <p>The Tor Browser team signs Tor Browser releases. Import its key
-    (0x4E2C6E8793298290) by starting <i>cmd.exe</i> and typing:</p>
-    <pre>gpg.exe --keyserver pool.sks-keyservers.net --recv-keys 0x4E2C6E8793298290</pre>
-    <p>After importing the key, you can verify that the fingerprint
-    is correct:</p>
-    <pre>gpg.exe --fingerprint 0x4E2C6E8793298290</pre>
+    </p>
+    <p>
+    The Tor Browser team signs Tor Browser releases. Import its key
+    (0x4E2C6E8793298290) by starting <i>cmd.exe</i> and typing:
```

Comment:
Always use long key ids. Short key ids are easy to fake.

Reply:
Isn't this the long id? It's the same as listed on our [signing-keys](https://www.torproject.org/docs/signing-keys.html.en) page.
```
$ gpg2 --keyid-format long --list-key 0x4E2C6E8793298290
pub   rsa4096/4E2C6E8793298290 2014-12-15 [C] [expires: 2020-08-24]
      Key fingerprint = EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
uid                 [ unknown] Tor Browser Developers (signing key) <torbrowser@torproject.org>
sub   rsa4096/EB774491D9FF06E2 2018-05-26 [S] [expires: 2020-09-12]
```

Reply:
Oh, ok, I usually use the full key id. But if that's what we use, that's ok.